Shop.BLL/Interfaces/IAuthService.cs

```diff
@@ -0,0 +1,8 @@
+using Shop.BLL.Common.DataTransferObjects.Users;
+
+namespace Shop.BLL.Interfaces;
+
+public interface IAuthService{ 
+    Task RegisterUserAsync(UserRequestRegistrationDto dto); 
```

Comment:
use full name of variable like UserRequestRegistrationDto userRequestRegistrationDto